_topics/en/vaults.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Vaults
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Scripts and Addresses
+  - Security Enhancements
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Vaults** are a type of covenant that requires two separate
```

Comment:
Might be better to have the covenant link up here instead of below

Reply:
I agree this would be better, but we can't have links in the excerpt (this is a restriction of Jekyll, not ours, although we could work around this if we had to).